README.md

```diff
@@ -48,6 +48,20 @@ nix profile install github:SpyHoodle/pridefetch
 pridefetch
 ```
 
+## Running on Arch Linux
+from the AUR
```

Comment:
Could we capitalize "from" and put an empty line before and after this line, like in "Running on NixOS" ?